src/components/Articles/SingleArticle.jsx

```diff
@@ -7,16 +7,39 @@ const SingleArticle = () => {
     const {article_id} = useParams();
     const [curArticle, setCurArticle] = useState({});
     const [isLoading, setIsLoading] = useState(true);
+    const [votes, setVotes] = useState();
+    const [voted, setVoted] = useState(false);
 
     useEffect(() => {
         setIsLoading(true);
         requests.getArticle(article_id)
         .then((data) => {
             setCurArticle(data);
+            setVotes(data.votes);
             setIsLoading(false);
         })
     },[])
 
+    const vote = () => {
+        let amount = 0
+        if (!voted){
+            document.getElementById('voteButton').innerHTML="unvote"
```

Comment:
You should not be using the DOM here. You have specifically used a framework that means you can change elements without it.